app/Driver.php

```diff
@@ -4,38 +4,43 @@
 
 use Illuminate\Database\Eloquent\Model;
 use Spatie\Activitylog\Traits\LogsActivity;
+
 class Driver extends Model
 {
     use LogsActivity;
-    protected static $logName = 'driver';  // Name for the log 
+
+    protected static $logName = 'driver';  // Name for the log
     protected static $logAttributes = ['*']; // Log All fields in the table // Log when a driver is Created or Updated
     protected static $logOnlyDirty = true; // Only log the fields that have been updated
 
-    const delimiter = '~$~';
-    static public function getAliases() {
+    private const DELIMITER = '~$~';
+    public static function getAliases()
+    {
         $alias_list = Driver::pluck('alias');
 
-        foreach($alias_list as $i => $alias) {
+        foreach ($alias_list as $i => $alias) {
             $arr = array();
-            $arr = explode(self::delimiter, $alias);
+            $arr = explode(self::DELIMITER, $alias);
```

Comment:
Is this thing case sensitive or no ?

Reply:
All references were changed to uppercase. PSR12 mentions that as the standard for constants.